### ISSUE TITLE
Stucks on "Browser successfully launched", then fails 

### ISSUE THREAD

**Issue by msbrdmr** [fawazahmed0/youtube-uploader#251]
Launching browser...
Browser successfully launched
TimeoutError: waiting for selector `input[type="password"]:not([aria-hidden="true"])` failed: timeout 60000ms exceeded
